pepper-apis/src/test/java/org/broadinstitute/ddp/route/ProfileRouteTest.java

```diff
@@ -187,7 +188,7 @@ public void testAddProfileMissingBody() throws Exception {
     }
 
     /**
-     * tests to make sure that throws 400 Error when given a preferred language not in table.
+     * tests to make sure that throws HttpStatus.SC_BAD_REQUEST Error when given a preferred language not in table.
```

Comment:
When writing javadoc, we should stick with the HTTP status code. The consumer of the documentation doesn't need to know what we're doing behind the curtains, and may not have the documentation for the `HttpStatus` class in context.

Reply:
This was side effect of search & replace , not intentional. will fix that

Reply:
Ah, understood- Thank you!